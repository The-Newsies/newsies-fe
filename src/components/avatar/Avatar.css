.avatarImg {
  width: 45px;
  border-radius: 100%;
}