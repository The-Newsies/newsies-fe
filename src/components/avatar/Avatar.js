import React from 'react';
import PropTypes from 'prop-types';

export default function Avatar({ image }) {
  return (
    <div>
      <img src={image} />
    </div>
  );
}

Avatar.propTypes = {
  image: PropTypes.string.isRequired
};
