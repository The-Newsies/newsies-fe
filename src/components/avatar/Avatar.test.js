import React from 'react';
import { shallow } from 'enzyme';
import Avatar from './Avatar';

describe('Avatar component', () => {
  it('renders Avatar', () => {
    const wrapper = shallow(<Avatar image={'image url'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
