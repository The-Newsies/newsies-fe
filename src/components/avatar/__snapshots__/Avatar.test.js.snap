// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component renders Avatar 1`] = `
<div>
  <img
    src="image url"
  />
</div>
`;
